docs/admin_api/user_admin_api.md

```diff
@@ -1155,3 +1155,27 @@ GET /_synapse/admin/v1/username_available?username=$localpart
 
 The request and response format is the same as the
 [/_matrix/client/r0/register/available](https://matrix.org/docs/spec/client_server/r0.6.0#get-matrix-client-r0-register-available) API.
+
+### Find a user based on their ID in an auth provider
+
+The API is:
+
+```
+GET /_synapse/admin/v1/auth_providers/$provider/users/$external_id
+```
+
+A response body like the following is returned:
+
+```json
+{
+    "user_id": "@hello:example.org"
+}
+```
```

Comment:
What is the response if there is no user with the given external_id?

Can you include HTTP status codes please?